crates/ruff_python_parser/Cargo.toml

```diff
@@ -14,7 +14,8 @@ build = "build.rs"
 [lib]
 
 [dependencies]
-ruff_python_ast = { path = "../ruff_python_ast"}
+ruff_python_ast = { path = "../ruff_python_ast" }
+ruff_python_trivia = { path = "../ruff_python_trivia" }
```

Comment:
Is the new dependency required because `is_python_whitespace` is now in `ruff_python_trivia`? It may make sense just to copy over the function (private) to avoid cyclic dependencies (ruff_python_trivia has a dev dependency on ruff_python_parser)